recipes/fastjet/meta.yaml

```
@@ -0,0 +1,63 @@
+{% set name = "fastjet" %}
+{% set version = "3.4.0.6" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/fastjet-{{ version }}.tar.gz
+  sha256: 86dea05ce7a030db9fe14e5bf1c0bc46351b6e439f9b9ef6b14a55ceefcc7b7a
+  patches:
+    - 0001-Don-t-overwrite-external-CXXFLAGS-and-LDFLAGS.patch
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  script: {{ PYTHON }} -m pip install . -vv
+  missing_dso_whitelist:
+    - $RPATH/libfastjet.so.0
+    - $RPATH/libfastjettools.so.0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make
+    - libtool
+    - automake
+    - autoconf
+    - swig
+  host:
+    - pip
+    - pybind11
+    - python
+    - cgal-cpp
+    - gmp
+  run:
+    - awkward >=2
+    - numpy >=1.13.3
+    - python
```

```suggestion
    - python >= 3.7
```

Add lower bound constraints consistent with those in the release (https://github.com/scikit-hep/fastjet/blob/d07c735ebc5976fb8ff8ed182ab87d7b42b0354a/setup.cfg#L37).